js/views/SurveyListPage.js

```diff
@@ -30,9 +33,23 @@ const SurveyListPage = React.createClass ({
     this.setState({
       dataSource: this.state.dataSource.cloneWithRows(Store.surveys),
     })
+
+    if (this.state.list.length === 0)
+      loadSurveyList({}, this.loadList)
   },
 
   /* Methods */
+  loadList(error, response){
+    if (error) {
+      console.warn(error)
+    } else {
+      this.setState({
+        isLoading: false,
```

Comment:
It looks like this variable isn't used. If we do make use of it, it needs to be set to false when the Store already has surveys loaded.